spotify-api.el

```diff
@@ -1,16 +1,17 @@
 ;; spotify-api.el --- Spotify.el API integration layer
 
-;; Copyright (C) 2014-2018 Daniel Fernandes Martins
+;; Copyright (C) 2014-2019 Daniel Fernandes Martins
 
 ;; Code:
 
+(defvar *spotify-user*         nil "Cached user object")
 (defvar *spotify-oauth2-token* nil "Cached OAuth2 token")
-(defvar *spotify-user* nil "Cached user object")
+(defvar *spotify-oauth2-ts*    nil "Unix timestamp in which the OAuth2 token was retrieved, used to trigger the token renewal process shortly after the token expiration")
```

Comment:
@danielfm Any reason not to renew the token just *before* it expires instead of just after?

Reply:
No particular reason, this is just how I got it to work. We can change this to see if it will break something.

Reply:
Oh, by the way, the comment is wrong. I'm renewing before the token expires.